src/internal/config/config.go

```diff
@@ -47,19 +47,25 @@ func Factory() *Config {
 
 // Credentials represent Easee API credentials.
 type Credentials struct {
-	AccessToken  string    `json:"accessToken"`
-	RefreshToken string    `json:"refreshToken"`
-	ExpiresAt    time.Time `json:"expiresAt"`
+	AccessToken           string    `json:"accessToken"`
+	RefreshToken          string    `json:"refreshToken"`
+	AccessTokenExpiresAt  time.Time `json:"accessTokenExpiresAt"`
```

Comment:
We need to keep the old field and add a task at the app startup which will populate the new field if possible (migration task).